app/fortunes

```diff
@@ -0,0 +1,9775 @@
+"... the educated person is not the person who can answer the questions, but
```

Comment:
If this is a file of text, we should probably include a `.txt` suffix on the filename so people don't confuse it for a binary file.  `fortunes.txt`